### PR TITLE
Create attack style panel

### PR DESCRIPTION
**Pull request by CurtisGreen**
# Attack Style Panel
image

Simple PR to add attack style panel, make the buttons interactable and store their values in the cookie. Doesn't currently change XP gains between attack/strength/defense as we still need to decide how we'll handle those in the future. Currently plan is to make all of the buttons on the dashboard (inventory, skills, etc.) clickable

### CODE REVIEW

File: src/scenes/load.js [hunk 1]
```diff
@@ -137,8 +137,29 @@ export class LoadScene extends Phaser.Scene {
...
         this.load.image("scroll-footer", "src/assets/ui/ScrollFooter.png");
         this.load.image("scroll-button", "src/assets/ui/ScrollButton.png");
+        this.load.image(
+            "attack-style-button",
+            "src/assets/ui/buttons/AttackStyleButton.png"
+        );
+        this.load.image("attack-style-panel", "src/assets/ui/AttackStylePanel.png");
+        this.load.image(
+            "attack-style-1-button",
+            "src/assets/ui/buttons//AttackStyle1Button.png"
```
Comment (AndrewVieira) [inline src/scenes/load.js]: There's two forward slashes here

**Review by AndrewVieira — COMMENTED** [review]
The code seems good to me. I tested it out and I had no issues. I like that the panel keeps the attack style the same when changing settings or refreshing the page.

Good work so far!

File: src/scenes/load.js (hunk 1 above)
Comment (CurtisGreen) [reply src/scenes/load.js]: Fixed https://github.com/EthanVieira/rs-clicker/pull/69/commits/602083f85d2aea2b594e1370226da5e3845838b1

**Review by EthanVieira — APPROVED** [review]
Nice job!

**Pull request merged**